cmd/payload-testing-ui/server.go

```diff
@@ -225,6 +241,27 @@ func (s *server) runDetails(w http.ResponseWriter, r *http.Request) {
 			ret := fmt.Sprintf(`<a href="https://github.com/%s/%s/commit/%s">%s</a>`, org, repo, h, h)
 			return template.HTML(ret)
 		},
+		"commentLink": func(obj *metav1.ObjectMeta, pr *prpqv1.PullRequest, config *prpqv1.ReleaseControllerConfig) template.HTML {
+			created := obj.GetCreationTimestamp().Format("Jan 2, 2006 15:04")
+			author := template.HTMLEscapeString(pr.Author)
+			ocp := template.HTMLEscapeString(config.OCP)
+			release := template.HTMLEscapeString(config.Release)
+			spec := template.HTMLEscapeString(config.Specifier)
+			ret := fmt.Sprintf(`%s's '/payload %s %s %s' on %s`, author, ocp, release, spec, created)
+			return template.HTML(ret)
+		},
+		"configLink": func(config *prpqv1.ReleaseControllerConfig) template.HTML {
+			ocp := template.HTMLEscapeString(config.OCP)
+			release := template.HTMLEscapeString(config.Release)
+			suffix := ""
+			if release == "ci" {
+				suffix = "-ci"
+			}
```

Comment:
Does this cover all types of `release-controller` configuration files currently in `openshift/release`?

This is not too bad, if it does we can live with it instead of loading and parsing those files.

Reply:
For what I saw it covers them. From 4.1 to 4.13